Make Dataset/Dockerfile

```diff
@@ -1,5 +1,4 @@
 FROM colmap/colmap:latest
```

Comment:
Why did you move this file? If it's common docker file for a repo it would be better to move it back or make another one for notebooks